pipelines/pipeline.yml

```diff
@@ -178,10 +178,10 @@ stages:
                       # so we'll just filter by status, and then
                       # filter by principalId client side
                       set +e
-                      leases=$(
-                        ./dce --config=dce.yml leases list -s Active \
-                        2>&1 >/dev/null)
```

Comment:
Previous versions for dce-cli output results to stderr,
so we needed this little `2>&1 >/dev/null` all over the place, to redirect from stderr to stdout.